shell/packages/sandstorm-ui-grainlist/grainlist-client.js

```diff
@@ -244,6 +244,63 @@ Template.sandstormGrainListPage.onRendered(function () {
 });
 
 Template.sandstormGrainListPage.events({
+  "click .grain-list .question-mark": function (event) {
+    const templateData = Template.instance().data;
+
+    const exitAndRemoveOverlayNow = () => {
+      // If there is no active intro, bail now.
+      if (!templateData.intro) {
+        return;
+      }
+
+      // Ask introjs to exit.
+      templateData.intro.exit();
+      // Remove our note-to-self which is how we detect the intro being around.
+      templateData.intro = null;
+      // Remove the overlay right now so that we can react to speedy clicking. introjs will do this
+      // itself, but queues the task to be done 0.5 seconds later. This is not very Meteoric, I
+      // realize.
+      const overlay = document.querySelector(".introjs-overlay");
+      if (overlay) {
+        overlay.remove();
+      }
+    };
+
+    if (templateData.intro) {
+      // In this case, the intro is currently active, and the user clicked on the question mark. The
+      // sensible thing to do is to dismiss the intro and stop processing the click.
+      exitAndRemoveOverlayNow();
+      return;
+    }
+
+    const intro = Template.instance().data.intro = introJs();
+    intro.setOptions({
+      steps: [
+        {
+          element: document.querySelector(".grain-list .question-mark"),
+          intro: "Each document, chat room, mail box, notebook, blog, or anything else you create is a grain. Grains are private until they are shared.",
```

Comment:
To me the missing information in the original phrasing is how they become "shared". What I want to express is that it is the grain owner's choice to share a grain. Sure, to most users this should be obvious but spelling it out means putting less cognitive load on them.

> I don't love "Your grains are private until you share them" because it's true of all grains,

I don't understand what you're saying here. Only your grains are private until you share them. Other people's grains are private until _they_ share them.